### PR TITLE
GNOME 45 Port

### LINKED ISSUE #401: GNOME 45
https://github.com/NickvisionApps/Proposals/issues/10

### PR DESCRIPTION
**Pull request by nlogozzo**
Closes #401 

Folder Open, No Files Found:
image

Folder Open, With Files, No Selection:
image

Folder Open, With Files, Selection:
image

Folder Open, With Files, Selection, Sidebar Collapsed:
image

### CODE REVIEW

**Comment by nlogozzo** [comment]
@DaPigGuy I think I did the new widgets right on MainWindow 😅

File: NickvisionTagger.GNOME/Views/MainWindow.cs [hunk 1]
```diff
@@ -1484,14 +1491,13 @@ private bool MusicLibraryUpdated()
             _applyAction.SetEnabled(false);
             _tagActionsButton.SetSensitive(false);
             _viewStack.SetVisibleChildName("Library");
-            _libraryFlap.SetFoldPolicy(_controller.MusicFiles.Count > 0 ? Adw.FlapFoldPolicy.Auto : Adw.FlapFoldPolicy.Always);
-            _libraryFlap.SetRevealFlap(true);
-            _flapToggleButton.SetSensitive(_controller.MusicFiles.Count > 0);
-            _filesViewStack.SetVisibleChildName(_controller.MusicFiles.Count > 0 ? "Files" : "NoFiles");
+            _librarySplitView.SetShowSidebar(_controller.MusicFiles.Count > 0 ? true : false);
```
Comment (DaPigGuy) [inline NickvisionTagger.GNOME/Views/MainWindow.cs:1494]:
```suggestion
            _librarySplitView.SetShowSidebar(_controller.MusicFiles.Count > 0);
```

**Review by DaPigGuy — CHANGES_REQUESTED** [review]
Just one nit

**Comment by DaPigGuy** [comment]
The sidebar doesn't seem to collapse properly on shrinking the window

**Comment by nlogozzo** [comment]
> The sidebar doesn't seem to collapse properly on shrinking the window due to a missing minimum width (`width-request`)

The sidebar has:
```
sidebar: Gtk.Box {
              width-request: 400;
```

**Comment by nlogozzo** [comment]
@DaPigGuy I realized what I was doing wrong...fixed in latest commit

**Pull request merged**